roles/openshift_cluster_monitoring_operator/files/cluster-monitoring-operator.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: openshift-cluster-monitoring-operator
+  annotations:
+    openshift.io/display-name: OpenShift Cluster Monitoring Operator
+    description: The OpenShift Cluster Monitoring Operator manages the deployment of the OpenShift monitoring platform.
+    iconClass: icon-openshift
+    tags: openshift,infra,monitoring
+    openshift.io/documentation-url: https://github.com/coreos-inc/tectonic-prometheus-operator/
+    openshift.io/support-url: https://access.redhat.com
+openshift.io/provider-display-name: Red Hat, Inc.
+parameters:
+- name: OPERATOR_IMAGE
+  value: quay.io/coreos/cluster-monitoring-operator:198d25d6
+- name: PROMETHEUS_OPERATOR_IMAGE
+  value: quay.io/coreos/prometheus-operator-dev
+- name: ALERTMANAGER_IMAGE
+  value: quay.io/prometheus/alertmanager
+- name: PROMETHEUS_IMAGE
+  value: quay.io/prometheus/prometheus
+- name: PROMETHEUS_CONFIG_RELOADER_IMAGE
+  value: quay.io/coreos/prometheus-config-reloader
+- name: CONFIG_RELOADER_IMAGE
+  value: quay.io/coreos/configmap-reload
+- name: NAMESPACE
+  # This namespace cannot be changed. Only `openshift-monitoring` is supported.
+  value: openshift-monitoring
+objects:
```

Comment:
This should probably also include the Alertmanager configuration secret as that's entirely to be configured by a user.

Reply:
TPO will create such secret if it doesn't exist, right? Is there any benefit of having a secret here rather than just letting TPO create it?

Reply:
The TPO creates a minimal working example if no secret exists, as the Alertmanager won't start otherwise, but for a declarative installs it's probably a good idea for users to specify their config through here.

Reply:
That's fine by me, but I don't think it should block merging this role to master. We can add that functionality in a separate pull request

Reply:
Sounds good to me as well